src/index.js

```diff
@@ -0,0 +1,69 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+import loaderUtils from 'loader-utils';
+
+export function pitch(remainingRequest) { // eslint-disable-line no-unused-vars
+  const query = loaderUtils.getOptions(this) || {};
+
+  let chunkNameParam = '';
+
+  if (query.name) {
+    const options = {
+      context: query.context || this.options.context,
+      regExp: query.regExp,
+    };
+    const chunkName = loaderUtils.interpolateName(this, query.name, options);
+    chunkNameParam = `, ${JSON.stringify(chunkName)}`;
+  }
+
+  let result;
+  if (query.lazy) {
```

Comment:
- Lazy references a common naming convention for `loading` of something. 
- Async is about order of execution or lack there of to be more specific.

Within this particular context i think `lazy` is the correct name for the query option. Given a bundle is either loader `lazy` or `not` I don't see the need to define a `sync` option given it's the only other practical alternative to `lazy`.